### PR TITLE
Configure skip-review label to enable auto-merge a few trusted PRs

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
To address https://github.com/knative/test-infra/pull/2729#issuecomment-816994968

/cc

### CODE REVIEW

File: config/prod/prow/jobs/custom/test-infra.yaml [hunk 1]
```diff
@@ -358,7 +358,43 @@ periodics:
     - name: flaky-test-reporter-slack-token
       secret:
         secretName: flaky-test-reporter-slack-token
-- cron: "15 15-23 * * 1-5"  # Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
+- cron: "15 15-23 * * 1-5"  # Bump with label `skip-review`. Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
```
Comment (chaodaiG) [inline config/prod/prow/jobs/custom/test-infra.yaml:361]: Considering how stable prow is, most of the time there is no problem merging every day. But to be more cautious, I might suggest bumping this following prow update schedule, which is Monday every week

**Review by chaodaiG — APPROVED** [review]
/hold

File: config/prod/prow/jobs/custom/test-infra.yaml (hunk 1 above)
Comment (chizhg) [reply config/prod/prow/jobs/custom/test-infra.yaml]: SGTM. Done.

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**